pkg/operator/discover/discover.go

```diff
@@ -82,7 +82,8 @@ func (d *Discover) Start(namespace, discoverImage string) error {
 }
 
 func (d *Discover) createDiscoverDaemonSet(namespace, discoverImage string) error {
-	privileged := false
+	// discovering the device uuid with sgdisk is required to run privileged
+	privileged := true
```

Comment:
does this have further impact on the security and openshift work you have been doing?

Reply:
it does mean the rook-ceph-system namespace will need to run privileged pods